test/celeritas/MockTestBase.cc

```diff
@@ -105,24 +105,29 @@ auto MockTestBase::build_particle() -> SPConstParticle
     using namespace units;
 
     constexpr auto zero = zero_quantity();
-    constexpr auto stable = ParticleRecord::stable_decay_constant();
 
     ParticleParams::Input inp;
-    inp.push_back({"gamma", pdg::gamma(), zero, zero, stable});
-    inp.push_back(
-        {"celeriton", PDGNumber{1337}, MevMass{1}, ElementaryCharge{1}, stable});
+    inp.push_back({"gamma", pdg::gamma(), zero, zero, stable_decay_constant});
+    inp.push_back({"celeriton",
+                   PDGNumber{1337},
+                   MevMass{1},
+                   ElementaryCharge{1},
+                   stable_decay_constant});
```

Comment:
Would've been fewer LOC changed if you just used `constexpr auto stable = stable_decay_constant;` 😉